cmd/vclusterctl/cmd/platform/add/cluster.go

```diff
@@ -70,7 +70,7 @@ vcluster platform add cluster my-cluster
 		},
 	}
 
-	c.Flags().StringVar(&cmd.Namespace, "namespace", "loft", "The namespace to generate the service account in. The namespace will be created if it does not exist")
```

Comment:
Can we test that this works as expected, I'm worried this will not work as we hardcode loft in the platform backend I believe

Reply:
Ok I went back and did a few more tests of this without specifying the namespace to get `vcluster-platform` as the default and tried out some random namespaces too.   Perhaps I'm misunderstanding something.  I didn't think the namespace here mattered much, it would just be the namespace of the deployment, to then make a management.loft.sh/v1 cluster object.  Can you point me to some places I should look in the platform to double check besides my just running the add command and creating vcluster instances in the external cluster(s)?

Reply:
I added the relevant logic